chapters/grid-search.qmd

```diff
@@ -0,0 +1,390 @@
+---
+knitr:
+  opts_chunk:
+    cache.path: "../_cache/grid/"
+---
+
+# Grid Search {#sec-grid}
+
+```{r}
+#| label: grid-setup
+#| include: false
+source("../R/_common.R")
+
+# ------------------------------------------------------------------------------
+
+library(kableExtra)
+library(lme4)
+library(broom.mixed)
+library(tidymodels)
+library(finetune)
+library(bonsai)
+library(patchwork)
+library(rlang)
+library(future)
+
+# ------------------------------------------------------------------------------
+
+source("../R/setup_two_class_example.R")
+
+set.seed(943)
+sim_tr <- sim_logistic(2000, sim_f)
+
+set.seed(14)
+sim_rs <- vfold_cv(sim_tr)
+
+set.seed(14)
+sim_nested_rs <- nested_cv(sim_tr, outside = vfold_cv(), inside = vfold_cv())
+
+load("../RData/grid_benchmarks.RData")
+
+# ------------------------------------------------------------------------------
+# Set Options
+
+plan("multisession")
```

Comment:
The latest version of tune suggests using the future package to setup the parallel workers.